L14Jdbc/src/main/java/ru/otus/l14/jdbc/DbExecutorImpl.java

```diff
@@ -0,0 +1,131 @@
+package ru.otus.l14.jdbc;
+
+import ru.otus.l14.jdbc.annotations.Id;
+import ru.otus.l14.jdbc.exceptions.DbExecutorException;
+
+import java.lang.reflect.Field;
+import java.sql.*;
+import java.util.*;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.*;
+import static ru.otus.l14.jdbc.SqlManager.QueryType.*;
+
+public class DbExecutorImpl<T> implements DbExecutor<T> {
+
+    private List<Field> fields = new ArrayList<>();
+
+    public static <T> DbExecutorImpl<T> of(Class<T> clazz) {
+        DbExecutorImpl<T> executor = new DbExecutorImpl<>();
+        executor.getClassFields(clazz);
+        return executor;
+    }
+
+    private DbExecutorImpl() {
+    }
+
+    @Override
+    public void create(T objectData) throws DbExecutorException {
+        PrimaryField primaryKey = this.getPrimaryKey(objectData);
+
+        try (Connection connection = SqlManager.getConnection();
+             PreparedStatement statement = connection.prepareStatement(SqlManager.generateQuery(INSERT, objectData.getClass().getSimpleName(), this.fields), Statement.RETURN_GENERATED_KEYS)) {
+
+            List<Field> fieldsWithoutId = this.fields.stream().filter(field -> !field.getName().equals(primaryKey.field.getName())).collect(toList());
```

Comment:
Можно выделить в метод. К тому же этот код используется еще в другом месте.
```suggestion
            List<Field> fieldsWithoutId = getFieldsWithoutId(...);
```